scipy/sparse/linalg/isolve/iterative.py

```diff
@@ -1,9 +1,10 @@
 """Iterative methods for solving linear systems"""
 
-__all__ = ['bicg','bicgstab','cg','cgs','gmres','qmr']
+__all__ = ['cg','cgs','icgs','bicg','bicgstab','cgnr','cgne','gmres','qmr']
 
 import warnings
 import numpy as np
+import time
```

Comment:
Looks like a left-over from development work, can you remove it?

Reply:
Ok, the `time` module was used to check performance in real time when I use SciPy to solve some physical field equations from PDEs, I'll remove it, thanks.